src/test/java/org/broadinstitute/consent/http/db/DataAccessRequestDAOTest.java

```diff
@@ -762,6 +779,40 @@ void testFindAllDataAccessRequestDatasArchived() {
     assertEquals(1, returnedDAR.size());
   }
 
+  @Test
+  void testFindDARsByDateRange() throws InterruptedException {
+    DarCollection darCollection = createDarCollection();
+    darCollection.getDars().keySet().forEach((referenceId) -> {
+      dataAccessRequestDAO.updateDraftToSubmittedForCollection(darCollection.getDarCollectionId(),
+          referenceId);
+    });
```

Comment:
A one-line lambda doesn't need `{}`s
```suggestion
    darCollection.getDars().keySet().forEach((referenceId) ->
        dataAccessRequestDAO.updateDraftToSubmittedForCollection(darCollection.getDarCollectionId(), referenceId));
```